doc/user/content/sql/functions/cast.md

```diff
@@ -31,29 +31,92 @@ The following special syntax is permitted if _val_ is a string literal:
 
 ### Valid casts
 
-Source type | Return type
-------------|------------
-`bytea`  | `text`
-`int` | `boolean`
-`int` | `float`
-`int` | `decimal`
-`int` | `text`
-`float`| `int`
-`float`| `decimal`<sup>1</sup>
-`float`| `text`
-`decimal` | `int`
-`decimal` | `float`
-`decimal` | `text`
-`date` | `timestamp`
-`date` | `timestamp with time zone`
-`date` | `text`
-`list` | `text`
-`time` | `interval`
-`time` | `text`
-`interval` | `time`
-`interval` | `text`
-`text`  |  `bytea`
-`text` | `list`
+Casts may be:
+
+* **Implicit** -- Values are automatically converted (for example, when you add `int4` to `int8`, the `int4` value is automatically converted to `int8`)
+* **Assignment** -- Values of one type are converted automatically when inserted into a column of a different type
+* **Explicit** -- You must invoke `CAST` deliberately
+
+Casts allowed in less strict contexts are also allowed in stricter contexts: that is, implicit casts can also occur explicitly or by assignment, and casts by assignment can also be explicitly invoked.
```

Comment:
This is implicit in the review above, but this notion of strictness and ordering should be more explicit

Reply:
Isn't that covered by

> Casts allowed in less strict contexts are also allowed in stricter contexts. That is, implicit casts can also occur explicitly or by assignment, and casts by assignment can also be explicitly invoked.